ui/shared/StyledTextArea.qml

```diff
@@ -50,10 +50,10 @@ Item {
             id: textArea
             text: ""
             font.pixelSize: 15
-            wrapMode: Text.WordWrap
+            wrapMode: Text.WrapAnywhere
```

Comment:
What do you think if we replace anywhere where we use `WordWrap` and `WrapAnywhere` to avoid words being cut in the middle and also to avoid text exceeding content width. 

> Text.Wrap - if possible, wrapping occurs at a word boundary; otherwise it will occur at the appropriate point on the line, even in the middle of a word.